DatadogWebViewTracking/Sources/WebViewMessage.swift

```diff
@@ -6,10 +6,10 @@
 
 import Foundation
 
-internal typealias JSON = [String: Any]
+public typealias JSON = [String: Any]
 
 /// Intermediate type to parse WebView messages and send them to the message bus
-internal enum WebViewTrackingMessage {
+public enum WebViewMessage {
     /// A log message with a JSON payload
     case log(JSON)
```

Comment:
suggestion/ Move `JSON` to `WebViewMessage` typealias. Otherwise we expose so popular type in public that it can commonly clash with user code.